app/src/components/market/common/warning_message/index.tsx

```diff
@@ -47,6 +47,7 @@ interface Props {
   additionalDescription: string
   danger?: boolean
   grayscale?: boolean
+  style?: React.CSSProperties
```

Comment:
Any reason for adding this prop if it's not being used?

Reply:
@KadenZipfel , it's necessary to add "margin" using inline style.
I can see that warning component is used several times on the project, and need to set different margin,
So I added it